Formula/tmux.rb

```diff
@@ -32,11 +34,16 @@ class Tmux < Formula
   def install
     system "sh", "autogen.sh" if build.head?
 
+    configure_args = [
+      "--disable-Dependency-tracking",
+      "--prefix=#{prefix}",
+      "--sysconfdir=#{etc}",
+    ]
```

Comment:
I definitely don't want to ignore you when your intent was to improve me and the codebase. I'm sorry if the way I asked the questions implied an abrasive or confrontational tone, definitely not my intent.

I think I also misunderstood your comment regarding the `args` argument. The two formula I used for inspiration actually used the `[]` format, which sounds more like bad luck on my part.